lib/environment.rb

```diff
@@ -267,8 +267,8 @@ def system_docker_repo
     # helper parser
     def parse_version(ver_str)
       ver = ver_str.sub(/(^v|^openshift-clients-)/,"")
-      if ver !~ /^[\d.]+$/
-        raise "version '#{ver}' does not match /^[\d.]+$/"
+      if ver !~ /^[\d][.][\d].*$/
+        raise "version '#{ver}' does not match /^[\d][.][\d].*$/"
```

Comment:
What do you think about `/^\d[.]\d+\b.*$/`? This will require one digit, then dot, then any number of digits, then a word boundary followed by anything.

Reply:
@akostadinov Updated.